### DISCUSSION TITLE
能够从知网抓取条目，但是无法对已有的pdf文件抓取知网元数据

### DISCUSSION THREAD

**Discussion by dawnjung7** [l0o0/jasminum#276] [Q&A]
您好，我的zotero版本是7.0.0-beta.77+adaa61f2c (64-bit)，jasminum的版本是1.0.0-16，目前我可以从知网中抓取条目，但是已经下载好的pdf始终无法成功抓取知网元数据。
请问这种情况如何解决？非常感谢！
![1715675043478](https://github.com/l0o0/jasminum/assets/92716949/33290e4b-7719-4e06-a657-f4a12384686d)
![image](https://github.com/l0o0/jasminum/assets/92716949/9f28db8f-3508-41fb-8403-0258ba246ee6)
![image](https://github.com/l0o0/jasminum/assets/92716949/1b10f0d4-041f-415b-a156-3d477c3db4c6)

**Comment by jiaojiaodubai** [comment]
对已有的pdf文件抓取基于文件名包含的标题，如果标题不正确或者知网确实搜不到，就无法抓取，同时也跟网络环境有关

**Comment by Wangyi20** [comment]
我也是这个问题，请问有解决吗。

**Reply by jiaojiaodubai** [reply]
见上一条回复

**Reply by Wangyi20** [reply]
感谢感谢，我的网络测试过没问题的。
回复楼主，这个问题我解决了，更新20版本的插件后，pdf拖进去后就能抓到了。
![微信图片_20241216120108](https://github.com/user-attachments/assets/bc62311a-031d-4774-8985-9072f212f2eb)
